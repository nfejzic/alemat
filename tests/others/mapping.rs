use alemat::{
    elements::{grouping::Row, Frac, Ident},
    MathMl,
};

#[test]
fn map_in_row() {
    let out = MathMl::with_content(
        Frac::builder()
            .num(Ident::from("x"))
            .denom(Ident::from("y"))
            .build(),
    )
    .map(Row::from)
    .render();

    crate::snap_test!(out, name: "others_map_in_row");
}
