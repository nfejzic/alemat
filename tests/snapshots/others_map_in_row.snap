---
source: tests/others/mapping.rs
expression: input
---
<math>
  <mrow>
    <mfrac>
      <mi>
        x
      </mi>
      <mi>
        y
      </mi>
    </mfrac>
  </mrow>
</math>

